src/main/java/mate/jdbc/dao/CarDao.java

```diff
@@ -0,0 +1,17 @@
+package mate.jdbc.dao;
+
+import java.util.List;
+import java.util.Optional;
+import mate.jdbc.model.Car;
+
+public interface CarDao {
+    Car create(Car car);
+
+    Optional<Car> get(Long id);
+
+    List<Car> getAll();
+
+    Car update(Car car);
+
+    boolean delete(Long carId);
+}
```

Comment:
You need to implement one more method (written in readme.md)
```suggestion
    boolean delete(Long carId);
    
    List<Car> getAllByDriver(Long driverId);
}
```

Reply:
давай додамо даний метод getAllByDriver до рівня ДАО, чому описала нижче